source/retryable-reads/tests/gridfs-download-serverErrors.yml

```diff
@@ -0,0 +1,152 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "files"
+
+data:
+    files:
```

Comment:
Shouldn't this be `fs.files` and `fs.chunks`? I'm pretty sure the GridFS spec tests assume the prefixes are added when fixtures are loaded, but this is a different test runner.

As for the collection name above, I wonder if you should introduce the concept of `bucket_name` and use `object: gridfsbucket` below. Otherwise, these GridFS test files seem quite inconsistent with other files in this spec.

Reply:
On the off chance this is a sensible improvement for the GridFS spec tickets (along with `object: gridfsbucket` and `bucket_name` at the top-level), feel free to open a SPEC ticket for GridFS so we can follow up with those improvements down the line.